guillotina/renderers.py

```diff
@@ -17,6 +18,8 @@ def guillotina_json_default(obj):
             return str(obj)
     elif isinstance(obj, complex):
         return [obj.real, obj.imag]
+    elif isinstance(obj, datetime):
+        return obj.isoformat()
```

Comment:
@lferran what was the problem?

Reply:
@masipcat this was failing

https://github.com/plone/guillotina/pull/963/files#diff-eafa441162c17595b0c457d8d08e332eR13

Reply:
I think this is expected because we removed datetime from `guillotina_json_default` because we replaced `json`/`ujson` with `orjson`, that knows how to serialize datetime objects to json (https://github.com/ijl/orjson#datetime)

I supposed that orjson would be faster because is written in rust and in that case, we wouldn't need to run the `guillotina_json_default` probably is slow (is pure-python)